docs/identity-platform/permissions-consent-overview.md

```diff
@@ -81,6 +81,8 @@ Administrators can grant consent for themselves or for the entire organization.
 
 Authentication requests are prompted for admin consent if consent wasn't granted and if one of those high-privilege permissions is requested.
 
+Custom application scopes are not considered high-privilege for administrator consent authentication request purposes.
```

Comment:
I'm not quite sure I understand what you mean by custom application scopes. Do you mean scopes that aren't part of Microsoft Graph scopes?